Vagrantfile

```diff
@@ -71,15 +72,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       --foreman-db-type='postgresql' --foreman-db-database='foreman' --foreman-db-host='pg1.localdomain' \
       --foreman-db-manage=false --foreman-db-username='foremandbuser' --foreman-db-password='Y3ll0-h@t' \
       --puppet-dns-alt-names='puppet.localdomain' \
-      --foreman-passenger-interface='172.28.128.20'
+      --foreman-passenger-interface='172.28.128.20' ; true
```

Comment:
This is a temporary stop gap to force the command to exit with a safe exit code because the foreman-installer is going to fail to start the Puppet server regardless. It is temporarily being fixed with the `sed` line until they update theforeman-installer.